src/locale/en.json

```diff
@@ -129,6 +130,24 @@
         "lead": "We fully understand that the software has to solve business problems: optimize tasks and resources, secure data, enable further scaling, and make life easier. Nowadays, most or practically all processes taking place in companies are often based on IT systems. We want to help make sure everything works as well as possible."
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "pwsz": {
```

Comment:
Maybe we should call it "cwup" or "witelonka"?

Reply:
changed to `cwup`